src/app/audiologist-login/audiologist-login.component.ts

```diff
@@ -33,8 +33,8 @@ export class AudiologistLoginComponent {
     if (this.audiologistUserName === 'Candi' && this.audiologistID === '123456') {
       Utilities.setSessionStorage('audiologist-pin', this.audiologistID);
       console.log('Audiologist log in ' + this.audiologistID);
-      Utilities.setSessionStorage('permissions' , 'audiologist');
```

Comment:
Rebase onto `development`, the new login functionality connects to the DB and can tell who is an audiologist and who is an admin.

Reply:
Done